ports/stm32/powerctrl.c

```diff
@@ -952,6 +973,9 @@ void powerctrl_enter_standby_mode(void) {
     PWR->CSR1 |= PWR_CSR1_EIWUP;
     #endif
 
+    // Disable Debug MCU.
+    DBGMCU->CR = 0;
```

Comment:
What is the reason for this change?  This affects all MCUs.

Reply:
This change seemingly goes against what you added in 32d76e5de66237eba456975095f998434413467a, where the debug module is enabled in low-power modes.

We should support debugging in low power modes if the user wants to do that.

Reply:
> This change seemingly goes against what you added in [32d76e5](https://github.com/micropython/micropython/commit/32d76e5de66237eba456975095f998434413467a), where the debug module is enabled in low-power modes.

Not really, that change is to make sure that debugging keeps working if/when the processor executes something like `WFI`, I think the debugger disconnected at that point before the processor woke up. The change here in this PR disables debug MCU on entry to standby intentionally so it doesn't draw any excess current. It doesn't really make sense to debug the MCU while it's in standby mode, it's going to reset on exit anyway, but I added a `#if defined(NDEBUG)`, so it's not executed on debug build, and you can remove this whole block it's just an extra precaution.

Reply:
OK, let's leave it in with the NDEBUG guard.